.github/workflows/ca-clone-test.yml

```diff
@@ -454,6 +630,73 @@ jobs:
 
           diff expected actual
 
+      - name: Check replica range config in secondary CA after cloning
+        if: always()
+        run: |
+          tests/ca/bin/ca-replica-range-config.sh secondary | tee output
+
+          # secondary range should remain 98-100
+          # next secondary range should be 1000-1099 initially
+          # 5 numbers were transferred to tertiary range
+          # so now next secondary range should be 1000-1094
+          cat > expected << EOF
+          dbs.beginReplicaNumber=98
+          dbs.endReplicaNumber=100
+          dbs.nextBeginReplicaNumber=1000
+          dbs.nextEndReplicaNumber=1094
```

Comment:
Since the increment is 100, I was expecting the next range to be 101 to 195. Are these value correct? The same for all the other tests.

Reply:
The `nextRange` for replica ID is actually initialized to `1000` here:
https://github.com/dogtagpki/pki/blob/master/base/ca/database/ds/create.ldif#L141
https://github.com/dogtagpki/pki/blob/master/base/kra/database/ds/create.ldif#L98
I updated the comments in the test to point to these files.

I'm not sure why it's done that way, but that's how it works now. This test is basically just recording the current behavior. Eventually we want DS replication to be set up separately from PKI installation so we don't have to deal with replica ID assignment anymore. In IPA case they do that already so they're not relying on this functionality.